src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -34,17 +34,40 @@
 				:disabled="isReadOnlyStateLoading"
 				@change="toggleReadOnly">
 			<label for="moderation_settings_lock_conversation_checkbox">{{ t('spreed', 'Lock conversation') }}</label>
+
+			<h3>{{ t('spreed', 'Limit writing to a conversation') }}</h3>
```

Comment:
please integrate with "lock conversation" which is another wording for the read-only state